samples/config_helper_vulkan.cc

```diff
@@ -609,6 +612,27 @@ std::string deviceTypeToName(VkPhysicalDeviceType type) {
   return "unknown";
 }
 
+std::string stageFlagBitsToNames(const VkShaderStageFlags bits) {
+
+    std::string result;
+    static const std::pair<VkShaderStageFlagBits, std::string> stages[] = {
+        std::make_pair(VK_SHADER_STAGE_VERTEX_BIT, "vert"),
+        std::make_pair(VK_SHADER_STAGE_TESSELLATION_CONTROL_BIT, "tessc"),
+        std::make_pair(VK_SHADER_STAGE_TESSELLATION_EVALUATION_BIT, "tesse"),
+        std::make_pair(VK_SHADER_STAGE_GEOMETRY_BIT, "geom"),
+        std::make_pair(VK_SHADER_STAGE_FRAGMENT_BIT, "frag"),
+        std::make_pair(VK_SHADER_STAGE_COMPUTE_BIT, "comp")
+    };
+    std::string separator;
+    for (const auto& stage : stages) {
+        if ((bits & stage.first) != 0) {
+            result += separator + stage.second;
+            separator = ", ";
```

Comment:
I'd prefer if this was a boolean to track if this is the first one emitted or not and add the ', ' if it isn't.

Reply:
changed to a bool